cscs-checks/tools/profiling_and_debugging/gperftools_mpi_omp.py

```diff
@@ -88,3 +99,12 @@ def setup(self, environ, partition, **job_opts):
         self.build_system.cxxflags = flags
         self.build_system.fflags = flags
         self.build_system.ldflags = flags + ['`pkg-config --libs libprofiler`']
+
+# {{{
+    @property
+    @sn.sanity_function
+    def report_flat_pctg(self):
+        regex = r'^\s+\d+ms\s+(?P<pctg>\d+.\d+)%.*_jacobi.\w+:\d+'
+        result = sn.extractsingle(regex, self.rpt_file_txt, 'pctg', float)
+        return result
+# }}}
```

Comment:
What is this?

Reply:
https://www.linux.com/tutorials/vim-tips-folding-fun/